spec/models/representation_order_spec.rb

```diff
@@ -52,13 +53,35 @@
         expect(representation_order.errors[:maat_reference]).to eq( [ 'invalid'])
       end
 
-      it 'should not error if 7-10 numeric digits' do
-        representation_order.maat_reference = '2078352232'
+      it 'should not error if 7 numeric digits' do
+        representation_order.maat_reference = '5078332'
         expect(representation_order).to be_valid
       end
+
+      context 'our test MAAT reference' do
+        before { representation_order.maat_reference = '2320006' }
+
+        context 'when on the live environment' do
+          before do
+            allow(ENV).to receive(:[]).with('MAAT_REGEXP').and_return /^[4-9][0-9]{6}$/
+            allow(Settings).to receive(:maat_regexp).and_return ENV['MAAT_REGEXP']
+          end
+
```

Comment:
just to break the tests down i'd be tempted to have a 

it 'uses environment configured MAAT regex' do
...
end